### ISSUE TITLE
question: pre defined filters for issues

### ISSUE THREAD

**Issue by benzkji** [taigaio/taiga-back#1613]
for me (and even more, new, onboarding users that use taiga), filters in the issues section are very important. I wonder if it is possible to predefine some filtersets for a project? like "somehow open", "in progress", etc, so users get used to, and dont have to dive into filters every time..or, every one of them, define filters for him/herself.

**Comment by miryamgduque** [comment]
Hi Benzkji,
At the moment, the only way to do this is for each user to create and save the custom filters that interest them.
We'll be evaluating a way to share custom filters among team members as it seems like it might be helpful.

If we introduce this new feature in our roadmap we'll let you know. Thanks for your feedback!

**Comment by benzkji** [comment]
was just onbarded to Jira, it has it...besides that it is horrible complex tool ;-)

thanks for your reply!

**Comment by miryamgduque** [comment]
I hope taiga fits you better 😉  If not, please let us know.

**Comment by benzkji** [comment]
yes, definitly, I prefer taiga. pre defined filters, and maybe some "cross project magic", like moving issues between projects, or acquiring stats over a set of projects...but...rants on a high level!

**Comment by benzkji** [comment]
would you accept a PR for this feature?
